src/tptp_parser.rs

```diff
@@ -181,7 +545,148 @@ pub fn parse_file<'a>(file: PathBuf) -> TPTPProblem {
     TPTPProblem {
         axioms: axioms,
         conjectures: conjectures,
-        negated_conjectures: negated_conjectures,
+    }
+}
+
+// Transform the TPTPProblem into the problem in CNF for our saturation prover:
```

Comment:
The transformation described here seems incorrect to me, consider that we want to prove:
```
(A1 & A2) -> (G1 & G2)
```
You would build
```
A = A1 & A2
G = ~G1 & ~G2
A & G
```
so in other words
```
A1 & A2 & ~G1 & ~G2
```
Now going manually we want:
```
~((A1 & A2) -> (G1 & G2))
~(~(A1 & A2) | (G1 & G2))
(A1 & A2) & ~(G1 & G2)
```
If we pull the not inside that yields
```
(A1 & A2) & (~G1 | ~G2)
```